### PR TITLE
Update Getting Started to Learn tutorials

### PR DESCRIPTION
**Pull request by im2nguyen**
### Acceptance tests
- [ ] Have you added an acceptance test for the functionality being added?
- [ ] Have you run the acceptance tests on this branch? (If so, please include the test log in a gist)

### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### CODE REVIEW

File: website/docs/guides/getting-started.html.markdown [hunk 1]
```diff
@@ -12,6 +12,9 @@ description: |-
 
 ## Kubernetes
 
+-> Visit the [Manage Kubernetes Resources via Terraform](https://learn.hashicorp.com/tutorials/terraform/kubernetes-provider?in=terraform/kubernetes&utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE) Learn tutorial for an updated getting
```
Comment (aareet) [inline website/docs/guides/getting-started.html.markdown:15]: Should we remove the source tracking from the URL and just have "https://learn.hashicorp.com/tutorials/terraform/kubernetes-provider?in=terraform/kubernetes" ? It's probably misleading to analytics

Comment (aareet) [reply website/docs/guides/getting-started.html.markdown:15]: Regarding the language, the current phrasing sounds a bit like this guide is an "outdated" one and the Learn tutorial is the updated one. Is the intention to eventually replace this guide with Learn?

**Comment by aareet** [comment]
Markdown looks good to me though.
Screen Shot 2020-09-01 at 2 59 41 PM

File: website/docs/guides/getting-started.html.markdown (hunk 1 above)
Comment (Redeux) [reply website/docs/guides/getting-started.html.markdown:15]: Yes, I agree with @aareet about the word "updated."  Perhaps "interactive" would be a better description?

**Pull request merged**